deso/events.go

```diff
@@ -424,17 +424,9 @@ func (node *Node) handleBlockConnected(event *lib.BlockEvent) {
 	// don't have a snapshot. We output extra metadata for this block to ensure
 	// Rosetta connects it appropriately.
```

Comment:
Removing this is a big deal. If I recall correctly this PutUtxoOps call here was preventing HyperSync from working at all for Rosetta on mainnet. The reason is that it was such a ginormous write that it would just stop the world and never finish, even with Performance badger params. And forget about using non-performance params.

Reply:
Hmmm maybe I'm remembering incorrectly. It seems like handleSnapshotCompleted may be the bottleneck to being able to run HyperSync. But I'm not sure now.

Another thing is I'm not sure this optimization will work because handleSnapshotCompleted dumps *different* historical UtxoOps for blocks into the index than what you would read from the chain db. So I think it will fail reconciliation because it will read the wrong "historical balance" prior to the snapshot height.